examples/uidl-samples/cms-project.json

```diff
@@ -2555,121 +1848,257 @@
+                        "elementType": "DataProvider",
+                        "dependency": {
+                          "type": "package",
+                          "path": "@teleporthq/thq-react-components",
+                          "version": "https://github.com/teleporthq/thq-react-components.git",
+                          "meta": {
+                            "namedImport": true
+                          }
+                        },
+                        "renderPropIdentifier": "context_74uql5",
+                        "nodes": {
+                          "success": {
+                            "type": "element",
+                            "content": {
+                              "elementType": "fragment",
+                              "children": [
+                                {
+                                  "type": "element",
+                                  "content": {
+                                    "elementType": "text",
+                                    "semanticType": "h1",
+                                    "referencedStyles": {},
+                                    "abilities": {},
+                                    "style": {
+                                      "marginTop": {
+                                        "type": "dynamic",
+                                        "content": {
+                                          "referenceType": "token",
+                                          "id": "--dl-space-space-oneandhalfunits"
+                                        }
+                                      }
+                                    },
+                                    "children": [
+                                      {
+                                        "type": "expr",
+                                        "content": "context_74uql5?.name"
+                                      }
+                                    ]
+                                  }
+                                },
+                                {
+                                  "type": "element",
+                                  "content": {
+                                    "elementType": "text",
+                                    "semanticType": "span",
+                                    "referencedStyles": {},
+                                    "abilities": {},
+                                    "children": [
+                                      {
+                                        "type": "expr",
+                                        "content": "context_74uql5?.id"
+                                      }
+                                    ]
+                                  }
+                                }
+                              ]
+                            }
+                          }
+                        },
+                        "valuePath": [],
+                        "itemValuePath": ["data"],
```

Comment:
We are not sending these values for the `cms-item` and `cms-list`. In Listing and details page. They are being handled using `exposeAs` attribute. So, this need to be corrected in the UIDL export from the platform